test/index.js

```diff
@@ -325,33 +384,4 @@ describe('hd-keyring', function() {
       })
     })
   })
-
-  describe('create and restore 1k accounts', function () {
```

Comment:
i realize upstream removed this test for some reason; is it possible for us to keep it?

Reply:
yes, added back in